<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>688</width>
    <height>693</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,1">
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="2" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>11</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1">
        <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,0,1">
         <item row="2" column="1">
          <widget class="QPushButton" name="allInactiveButton">
           <property name="text">
            <string>All Inactive</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="toggleAllButton">
           <property name="text">
            <string>Toggle All</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="0" rowspan="4">
          <layout class="QGridLayout" name="gridLayout" rowstretch="1,0">
           <item row="1" column="3">
            <widget class="LEDWidget" name="ledWidget3">
             <property name="color">
              <color>
               <red>0</red>
               <green>85</green>
               <blue>255</blue>
              </color>
             </property>
             <property name="state">
              <bool>true</bool>
             </property>
             <property name="flashing">
              <bool>false</bool>
             </property>
             <property name="flashRate">
              <number>181</number>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="VerticalTextLabel" name="ledOneLabel_2">
             <property name="styleSheet">
              <string notr="true">font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
             </property>
             <property name="text">
              <string>LED 2</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="LEDWidget" name="ledWidget2">
             <property name="color">
              <color>
               <red>0</red>
               <green>255</green>
               <blue>0</blue>
              </color>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="VerticalTextLabel" name="ledOneLabel_3">
             <property name="styleSheet">
              <string notr="true">font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
             </property>
             <property name="text">
              <string>LED 3</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="LEDWidget" name="ledWidget1"/>
           </item>
           <item row="0" column="1">
            <widget class="VerticalTextLabel" name="ledOneLabel">
             <property name="styleSheet">
              <string notr="true">font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
             </property>
             <property name="text">
              <string>LED 1</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="4">
            <spacer name="horizontalSpacer_6">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="0">
            <spacer name="horizontalSpacer_7">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="2" column="2">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="4" column="1" colspan="2">
          <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,1">
           <item>
            <widget class="QCheckBox" name="flashingCB">
             <property name="text">
              <string>Flashing</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Rate:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="rateSB">
             <property name="suffix">
              <string> ms</string>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>2000</number>
             </property>
             <property name="value">
              <number>200</number>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_8">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="allActiveButton">
           <property name="text">
            <string>All Active</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="2">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>125</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,0,0,1">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>IPv4 IP Address:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="IPAddressWidget" name="v4IPAddressWidget"/>
         </item>
         <item>
          <widget class="LEDWidget" name="validV4Indicator">
           <property name="color">
            <color>
             <red>0</red>
             <green>255</green>
             <blue>0</blue>
            </color>
           </property>
           <property name="state">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>122</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,0,0,1">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>IPv6 IP Address:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="IPAddressWidget" name="v6IPAddressWidget">
           <property name="addressType">
            <enum>IPAddressWidget::IPV6</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="LEDWidget" name="validV6Indicator">
           <property name="color">
            <color>
             <red>0</red>
             <green>255</green>
             <blue>0</blue>
            </color>
           </property>
           <property name="state">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>237</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0,1">
      <item>
       <widget class="NeedleGauge" name="needleGauge">
        <property name="value">
         <double>2415.000000000000000</double>
        </property>
        <property name="faceColor">
         <color>
          <red>180</red>
          <green>179</green>
          <blue>176</blue>
         </color>
        </property>
        <property name="marginSizeAsPercent">
         <number>2</number>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="fontColor">
         <color>
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </property>
        <property name="maxValue">
         <double>6000.000000000000000</double>
        </property>
        <property name="majorTickInterval">
         <number>10</number>
        </property>
        <property name="numberOfTicks">
         <number>100</number>
        </property>
        <property name="majorTickLengthAsPercent">
         <number>6</number>
        </property>
        <property name="minorTickLengthAsPercent">
         <number>4</number>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0">
        <item>
         <widget class="QSpinBox" name="needleValueSB">
          <property name="maximum">
           <number>8000</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="updateNeedleButton">
          <property name="text">
           <string>Update</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_9">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>688</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>IPAddressWidget</class>
   <extends>QWidget</extends>
   <header>IPAddressWidget.h</header>
  </customwidget>
  <customwidget>
   <class>LEDWidget</class>
   <extends>QWidget</extends>
   <header>LEDWidget.h</header>
  </customwidget>
  <customwidget>
   <class>NeedleGauge</class>
   <extends>QWidget</extends>
   <header>NeedleGauge.h</header>
  </customwidget>
  <customwidget>
   <class>VerticalTextLabel</class>
   <extends>QWidget</extends>
   <header>VerticalTextLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
